cli/src/action/time_interval.rs

```diff
@@ -0,0 +1,137 @@
+use crate::error::CliError;
+use std::convert::From;
+use std::fmt;
+use std::num::{ParseFloatError, ParseIntError};
+use std::time::Duration;
+#[derive(Copy, Clone)]
+pub struct TimeInterval {
```

Comment:
please add rust module documentation

Reply:
added some basic documentation, if there's a guide on how much to include I would love to see it